ingestion-edge/ingestion_edge/config.py

```diff
@@ -49,3 +49,17 @@ class Route:
 QUEUE_PATH = environ.get("QUEUE_PATH", "queue")
 
 MINIMUM_DISK_FREE_BYTES = int(environ.get("MINIMUM_DISK_FREE_BYTES", 0)) or None
+
+METADATA_HEADERS = dict(
+    json.loads(
+        environ.get(
+            "METADATA_HEADERS",
+            '{"Content-Length":"content_length",'
+            '"DNT":"dnt",'
+            '"Date":"date",'
+            '"User-Agent":"user_agent",'
+            '"X-Forwarded-For":"x_forwarded_for",'
+            '"X-Pingsender-Version":"x_pingsender_version"}',
+        )
+    )
+)
```

Comment:
I'd prefer a triple-quoted string here to make this more readable. Perhaps ideal would be breaking out the default:

```
METADATA_HEADERS_DEFAULT = '''
{
  "Content-Length": "content_length",
  ...
}
'''

METADATA_HEADERS = dict(json.loads(environ.get("METADATA_HEADERS", METADATA_HEADERS_DEFAULT)))
```

Reply:
i used a comma-separated string instead, and i feel like that addresses the readability. lmk if you still think it should be separated.

Reply:
That looks great.